packages/ember-views/lib/mixins/view_support.js

```diff
@@ -620,7 +620,7 @@ export default Mixin.create({
     this._super(...arguments);
 
     if (!this.elementId && this.tagName !== '') {
-      this.elementId = guidFor(this);
```

Comment:
There is a big difference, in that we want to isolate all of normalizing of component props to one place.  If this didn't work there there is something odd happening in the curly component manager that needs to be addressed.